Utilities/Scripts/python-package-updater.py

```diff
@@ -0,0 +1,77 @@
+# Use this script to update python packages to their latest version.
+# Use the latest Slicer preview's PythonSlicer.exe to run this script located in the Slicer/Utilities/Scripts directory
+# It will then locally update all the updated python versions with their updated hashes locally in the repo.
+# This script DOES NOT handle python package version incompatibilities or the addition/removal of other package dependencies for a given package.
+
+import subprocess
+import sys
+import requests
+import os
+import re
+
+if __name__ == '__main__':
```

Comment:
I suggest to move the code into its own function before `if __name__ == '__main__'` and then do:

```python

def update_external_project_python_packages(search_directory):
  [...]
  
if __name__ == '__main__':
  search_directory = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), "SuperBuild")
  update_external_project_python_packages(search_directory)
```